scripts/create_backup_2fa_codes.rb

```diff
@@ -0,0 +1,4 @@
+User.where(enabled_2fa: true).each do |user|
+  user.update(backup_2fa_code: SecureRandom.alphanumeric(24))
+  TwoFactorMailer.with(user: user, host: 'meta.codidact.com').backup_code.deliver_later

```

Comment:
Testing in development, with `deliver_later` it doesn't seem to deliver the emails (probably because there is no active job configuration). Is production configured in such a way that this does work?